Packs/CortexAttackSurfaceManagement/.pack-ignore

```diff
@@ -4,7 +4,7 @@ ignore=IF100
 [file:incidentfield-ASM_-_Playbook_Stage.json]
 ignore=IF100
 
-[file:Cortex_ASM_-_ASM_Alert.yml]
```

Comment:
@johnnywilkes This was intentionally removed?

Reply:
yes, that effectively disabled a validation error that was in Cortex_ASM_-_ASM_Alert.yml and got moved to Cortex_ASM_-_ServiceNow_Notification.yml